src/common/links.ts

```diff
@@ -1,6 +1,14 @@
 export const getDiscussionLink = (postId: string): string =>
   `${process.env.COMMENTS_PREFIX}/posts/${postId}`;
 
+const isRegexValidURL = (url: string) => {
```

Comment:
Why do we need both `isValidHttpUrl` and `isRegexValidUrl`?

Reply:
These are two ways to cover the validity of URLs. The `isValidHttpUrl` is the simplest way to validate but it fails in the case of `http://abc` which is not an actual valid URL. That case is caught by the regex.

I am using both just to be sure all edge-cases are covered

Reply:
This is kind of confusing. Future devs will not be able to understand it by name. Also, I wouldn't go into too much hassle of validating these URLs for this purpose.

Reply:
Cool. Then let me remove the validation through regex.